css/src/components/form/radio.scss

```diff
@@ -0,0 +1,96 @@
+$radio-circle-size: 1.125rem !default;
+$radio-dot-size: 0.75rem !default;
```

Comment:
have we used `em` sizing appropriately throughout the component?

Reply:
Originally when I used `em`, I thought that the radio was too small at `font-size-xs` so there's not a lot to click on. Upon second thought, you can also click on the label so perhaps it's a non-issue. Updated to em.